pom.xml

```diff
@@ -1226,7 +1226,16 @@
             <encoding>UTF-8</encoding>
             <maxmem>1024m</maxmem>
             <fork>true</fork>
+            <compilerId>javac-with-errorprone</compilerId>
```

Comment:
Can this be put in a profile? I'm slightly concerned about pushing the entire build through a different compiler just for this tool. It's easy to make this selectable.